extension/src/completion/IdCompletionProvider.ts

```diff
@@ -0,0 +1,30 @@
+import { CompletionProvider } from "./completionProvider";
+import * as vscode from "vscode";
+
+export class IdCompletionProvider implements CompletionProvider {
+
+    private tagName: string;
+    constructor(tagName: string) {
+        this.tagName = tagName;
+    }
+
+    public createSuggestion(document: vscode.TextDocument): vscode.CompletionItem[] {
+        const newId = this.getNewId(document);
+        return [new vscode.CompletionItem(newId.toString())];
+    }
+
+    private getNewId(document: vscode.TextDocument): number {
+        const regexToGetIds = new RegExp(`<${this.tagName}[\\s\\S]*? Id="(.*?)" [\\s\\S]*?>`, "g");
```

Comment:
does it work if we change order of parameters?

Reply:
Yes :)

Reply:
We have to more tests (I am not a regex expert).
But currently it is working pretty well